continuous_integration/scripts/build-legate

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set_repo_dir() {
+    set -xeuo pipefail
+
+    # Resolve the directory of the script
+    SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+    # Navigate to the parent of the parent of SCRIPT_DIR, then get the full path
+    REPO_DIR="$(cd "$SCRIPT_DIR/../.." && pwd)"
+
+    export REPO_DIR
+
+    export PATH="${PATH}:${REPO_DIR}/continuous_integration/scripts"
+    export ARTIFACTS_DIR="${REPO_DIR}/.artifacts"
+}
+
+# export CONDA_BUILD_THREADS=1
+# export JOBS=1
+
+build_legate_ci() {
+    build-legate-cpp;
+    build-legate-wheel;
+    build-legate-conda;
+}
+
+build_legate_release() {
+    # export NVCC_PREPEND_FLAGS='--verbose'
+
+    mkdir -p /tmp/env_yaml /tmp/conda-build
+
+    conda mambabuild --output-folder /tmp/conda-build -c nvidia -c conda-forge --no-include-recipe conda/conda-build
+}
+
+copy_release_artifacts() {
+    echo Copying release artifacts
+    cp /tmp/conda-build/linux-64/legate-core-*.tar.bz2 "$ARTIFACTS_DIR"
+}
+
+copy_ci_artifacts() {
+    echo Copying CI artifacts
+
+    cp -r /tmp/out          "$ARTIFACTS_DIR"
+    cp -r /tmp/conda-build  "$ARTIFACTS_DIR"
+}
+
+# NOTE: This script must be run from the project's root folder
```

Comment:
Not necessarily something that needs to be done as part of the PR, but if the script is in a known place, we can use the location of the script to get back to the root folder.

Reply:
> if the script is in a known place, we can use the location of the script to get back to the root folder.
Already doing this, just forgot to remove the comment. The comment has been removed now.